### PR TITLE
Updates Requisitions Pricing

### PR DESCRIPTION
**Pull request by Chaznoodles**
## About The Pull Request

Updates Requisitions pricing across the board, pulling information from Electric's previous Requisitions PR (#4269) to create what should be more accessible gear, yet still at a cost.

## Why It's Good For The Game

Tweaks pricings across the board to encourage use of some things which aren't used, and increased some pricings as a result of overuse. This should hopefully mean people will try things from Requisitions they haven't used before, and makes items more widely accessible.

## Changelog
:cl:
add: Larva embryos can now be sold to Requisitions. How much are you willing to forgo those precious advanced drugs?
add: New items added to Requisitions, including stealthy cloaks, armageddon in four doses and a method to send things skywards with ease. Sell the world.
del: Removed some unused Surplus weapons and regular weapons. We've got enough of those in the vendors.
tweak: Requisitions pricing tweaked across the board using information from Electric's previous PR to encourage buying things, but still at a cost.
tweak: Specialist gear is now cheaper, but doesn't come with ammunition. B17 and B18 have been dropped in price due to it piling up in the belowdecks. Please, we're drowning in it.
tweak: Miner crate prices have been upped somewhat, to account for a slower Requisitions points gain and slower crate generation.
tweak: Default ammunition in big ammo boxes upped from 600 to 800. Regular ammo boxes added to Requisitions listings.
/:cl:

### CODE REVIEW

**Comment by RenGusta** [comment]
Reminder;
Tweak ammo box volumes to bigger, because CM style volumes.

**Comment by Chaznoodles** [comment]
> Reminder;
> Tweak ammo box volumes to bigger, because CM style volumes.

@RenGusta Upped the default amount of loose ammo in ammo boxes from 600 to 800. We'll see if this has any effect and revisit at a later point.

**Comment by Chaznoodles** [comment]
thanks travis

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -679,53 +580,68 @@ ARMOR
...
+
+/datum/supply_packs/armor/nvg
+	name = "Night Vision Goggles"
+	contains = list(/obj/item/clothing/glasses/night/m42_night_goggles)
+	cost = 50
+
+/datum/supply_packs/armor/thermal
+	name = "Thermal Goggles"
+	contains = list(/obj/item/clothing/glasses/thermal/m64_thermal_goggles)
+	cost = 60
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:593]: No.

File: code/modules/reqs/supplypacks.dm [hunk 2]
```diff
@@ -34,13 +34,21 @@ OPERATIONS
...
 	name = "orbital beacon"
 	contains = list(/obj/item/squad_beacon/bomb)
 	cost = 30
 
+/datum/supply_packs/operations/fulton_extraction_pack
+	name = "fulton extraction pack"
+	contains = list(
+		/obj/item/fulton_extraction_pack,
+		/obj/structure/fulton_extraction_point
+	)
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:49]:
```suggestion
	name = "fulton extraction pack"
	contains = list(/obj/item/fulton_extraction_pack)
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/reqs/supplypacks.dm (hunk 2 above)
Comment (Chaznoodles) [reply code/modules/reqs/supplypacks.dm:49]: The fulton_extraction_point is a beacon to which you link the fulton, like how medevac beds work. From what I'm reading, you can fulton off xenos corpses without it for points, but this just gives another way to move stuff around. If people start using it en-masse as a cheaper medibed (if that's even possible), I'll punt it.

Comment (TiviPlus) [reply code/modules/reqs/supplypacks.dm:49]: It's a deprecated item, Fultons autosell their attached targets

Comment (Chaznoodles) [reply code/modules/reqs/supplypacks.dm:49]: based

File: code/modules/reqs/supplypacks.dm (hunk 1 above)
Comment (Chaznoodles) [reply code/modules/reqs/supplypacks.dm:590]: These were in before in Electric's PR. They make a difference, and their price reflects that. The former has a slightly bigger impact than a Baldur lamp, mesons and the like, while freeing up your module slot. The latter helps in tracking down those xenos that just keep running, and helps in stopping ambushes.

**Comment by Chaznoodles** [comment]
Reopening.

**Review by Chaznoodles — COMMENTED** [review]
go away

File: code/modules/reqs/supplypacks.dm (hunk 1 above)
Comment (Chaznoodles) [reply code/modules/reqs/supplypacks.dm:590]: @TiviPlus

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Comment by jroinc** [comment]
We should un-merge this so we can TM other things that require modifying cargo @Chaznoodles

**Pull request merged**